### PR TITLE
Develop: Metrics web-app(Covid-19 stats)

### PR DESCRIPTION
**Pull request by DanSam5K**
## In this project milestone I was able to accomplished the following:
- Adhere to general requirements by
    - [x] Making sure that there are no linter errors.
    - [x] Making sure that I used correct Gitflow.
    - [x] Made sure that you documented your work in a professional way.
    - [x] Also HTML/CSS & JavaScript best practices were followed
- Narrativa API: COVID-19 data was the chosen API and original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
- Home page features I implemented the following Interactions
     - [x] When the page loads, the webapp shows the list of categories that could be filtered out by some country name
     - [x] Along with the country name, I display total recorded cases values per country that come from the API. Then
         When the user clicks (or taps) on a country, the application navigates to the details page.
  - Details page I implemented the following Interactions
     - [x] In the details page, the webapp retrieves data from the API to show detailed data in the country.
     - [x] When the user clicks on the "Back" button (<), the user navigates to the home page.
  - Testing requirements
      - [x] I Create unit tests for pure functions in your Redux code.
      - [x] I Create integration tests for your application using the React Testing Library. By mocking the access to the API,  
   and connection to the Redux Store.
- Technically I made sure my application
   - [x] The project is a single page application (SPA) built with React and Redux.
   - [x] The data retrieved from the API was stored in the Redux store.
   - [x] I filter the data retrieve from the API using a Filter stateless component.
   - [x] Every page (the main page and the pages for each item)  have a unique route within the SPA.
   - [x] The project was deployed on netlify and accessible online also link was added to README. [click to view](https://cocky-williams-122084.netlify.app/).
- Project documentation
  - [x] I also added a [video](https://loom.com/share/f5b7840114e645999084a901c03c2eca) presenting the features of the project I built.

### CODE REVIEW

File: src/assets/css/App.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  background: #fb5092;
+}
+
+.bg-pink-light {
+  /* background-color: rgb(88, 137, 229) !important; */
+  background-color: #d6608e;
+}
+
+body {
+  /* background-color: rgb(62, 108, 194) !important; */
```
Comment (Dipeshtwis) [inline src/assets/css/App.css]: - [x] I would recommend removing the commented code to make the codebase cleaner.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @DanSam5K, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: App is deployed on netlify and working great.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/assets/css/App.css (hunk 1 above)
Comment (DanSam5K) [reply src/assets/css/App.css]: - Ooh it totally skipped my mind, I will fix that right away! Thank you for the honest feedback 👍

Comment (henatan99) [reply src/assets/css/App.css]: This issue is well addressed. Nice to make the requested changes. :clap: :+1:

**Review by henatan99 — APPROVED** [review]
Hi @DanSam5K , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] OPTIONAL: Your PR should have been from some feature branch to develop branch, making the develop branch the default branch. But you made a PR from develop branch to main. Kindly take note of this for your projects. :pray: 
- [x] OPTIONAL: You could have smaller components like card elements in the 'Home` component. Smaller components are more reusable, easier to fix, and more scalable. :pray: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DanSam5K** [comment]
### Hi @henatan99 ,
- That was what I did regarding working on different branch other than the development branch and then requesting for review to the development branch, but my PR was invalidated when try to request for review from the feature branch, but was ask to instead merge all my work from features branch to the development branch and request for a review from my development to main 👍 
- Meanwhile concerning refactoring I will improve on that significantly thank you.

**Comment by henatan99** [comment]
> ### Hi @henatan99 ,
> * That was what I did regarding working on different branch other than the development branch and then requesting for review to the development branch, but my PR was invalidated when request for review from the feature branch, but was ask to instead merge my all my features branch to the development branch and request for a review from my development to main +1

Hi @DanSam5K , so sorry, it was my mistake. I think they have updated the GitHub flow requirement. I revisited it and your GitHub flow is correct (from feature branch to main). :+1:

**Pull request merged**